### PR TITLE
f dist

### PR DESCRIPTION
**Pull request by Masha-Iris**
Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== RELEASE NOTES ==

General Changes
* F dist

Hive Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 1]
```diff
@@ -85,6 +85,18 @@ Mathematical Functions
     The a, b parameters must be positive real numbers and value v must be a real value.
     The value v must lie on the interval [0, 1].
 
+.. function:: inverse_f_cdf(n_df, d_df, p) -> double
```
Comment (rongrong) [inline presto-docs/src/main/sphinx/functions/math.rst:88]: Functions should be in alphabetical order.

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 2]
```diff
@@ -166,7 +178,7 @@ Mathematical Functions
 .. function:: truncate(x, n) -> double
 
     Returns ``x`` truncated to ``n`` decimal places.
-    ``n`` can be negative to truncate ``n`` digits left of the decimal point. 
+    ``n`` can be negative to truncate ``n`` digits left of the decimal point.
```
Comment (rongrong) [inline presto-docs/src/main/sphinx/functions/math.rst:181]: Please remove unrelated changes.

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 3]
```diff
@@ -85,6 +85,18 @@ Mathematical Functions
     The a, b parameters must be positive real numbers and value v must be a real value.
     The value v must lie on the interval [0, 1].
 
+.. function:: inverse_f_cdf(n_df, d_df, p) -> double
+
+    Compute the inverse of the F cdf with given numerator df, denominator df parameters for the cumulative
```
Comment (rongrong) [inline presto-docs/src/main/sphinx/functions/math.rst:90]: Please explain what `df` is in the documentation.

**Review by rongrong — COMMENTED** [review]
Please fix check style errors and test errors and squash all changes into a single commit (`git rebase -i`) and force push to update the pr (`git push -f origin BRANCH_NAME`)

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 4]
```diff
@@ -85,6 +85,18 @@ Mathematical Functions
...
+    probability (p): P(N < n). The numerator df, denominator df parameters must be positive real values.
+    The probability p must lie on the interval [0, 1].
+
+.. function:: f_cdf(n_df, d_df, v) -> double
```
Comment (talgalili) [inline presto-docs/src/main/sphinx/functions/math.rst:94]: please change v to value (also in other places in the doc)
